contracts/vaults/RibbonThetaVault.sol

```diff
@@ -536,26 +533,57 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         address newOption = nextOption;
         require(newOption > PLACEHOLDER_ADDR, "!nextOption");
 
+        uint256 pendingAmount = totalPending();
+        uint256 currentSupply = totalSupply();
+        uint256 currentBalance = assetBalance();
+        uint256 roundStartBalance = currentBalance.sub(pendingAmount);
+        uint16 currentRound = round;
```

Comment:
perhaps easier to remove `uint16 currentRound = round;` and do `round += 1;` after `roundPricePerShare[currentRound] = currentPricePerShare;`